src/index.js

```diff
@@ -2,20 +2,38 @@ import Tasks from './tasks.js';
 import './style.css';
 
 let id = 0;
-const tasksData = [new Tasks('Watch Movie', false, id += 1), new Tasks('Read a book', true, id += 1), new Tasks('Practice Algorithms', false, id += 1)];
 
+let tasksData = [new Tasks('Watch Movie', false, id += 1), new Tasks('Read a book', true, id += 1), new Tasks('Practice Algorithms', false, id += 1)];
+const setLocal = (() => {
+  localStorage.setItem('tasks', JSON.stringify(tasksData));
+});
+
+// eslint-disable-next-line consistent-return
 const showTasks = (() => {
   const tasksList = document.querySelector('.tasks');
+  if (tasksData === null) {
+    return null;
+  }
   tasksData.forEach((task) => {
     const li = document.createElement('li');
     const input = document.createElement('input');
     input.type = 'checkbox';
     input.checked = task.completed;
+    if (task.completed === true) {
+      li.classList.add('completed');
+    } else {
+      li.classList.remove('completed');
+    }
     input.addEventListener('change', () => {
       task.completed = input.checked;
       if (task.completed === true) {
         li.classList.add('completed');
+        task.completed = true;
+      } else {
+        li.classList.remove('completed');
+        task.completed = false;
```

Comment:
- [ ] It would be so much better if you write this action in a method and imported it as a module. Modules help you to develop and maintain the app easier running into minor bugs when you need to change or maintain the app. [Here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/lessons/lesson_es6_modules.md) is a reminder of modules. 

![Capture](https://user-images.githubusercontent.com/64173189/134580100-44a96464-69ce-475b-b3ed-3e2b830a26fd.JPG)

Please fix this minor issue 🛠️ 😄